flake.nix

```diff
@@ -15,11 +15,17 @@
       overlays.default = import ./overlay.nix;
       nix2nvimrcModules = readDirNix ./modules;
       nix2nvimrcConfigs = readDirNix ./configs;
+
+      nvimModules = map (i: i.file)
+        ((builtins.attrValues nix2nvimrcModules)
+          ++ (builtins.attrValues nix2nvimrcConfigs));
+      enableModules = map (i: i.enabler)
+        (builtins.attrValues nix2nvimrcConfigs);
     in
     {
       lib = import ./lib.nix;
 
-      inherit overlays nix2nvimrcModules nix2nvimrcConfigs;
+      inherit overlays nvimModules;
```

Comment:
Maybe better to provide two modules, like `nixosModules.abc`?
One `nvimModules.default`, which imports nix2nvimrc and configs
and one `nvimModules.config`, which contains only the configs from ck3d?